bin/racket_exercise_generator.py

```diff
@@ -0,0 +1,553 @@
+import json, toml, os, argparse, sys, string
+from custom_json_encoder import CustomJSONEncoder
+
+
+def find_racket_main():
+    up_one = os.path.split(os.getcwd())[0]
+    return ".." if os.path.split(up_one)[1] == "racket" else "."
+
+
+TARGET = os.path.abspath(find_racket_main() + "/exercises/practice")
+
+
+def create_directory_structure(exercise_name):
+    """
+    Creates the exercise, .meta, and .docs directories.
+
+    Parameter exercise_name: Name of the exercise to be generated.
+    Precondition: exercise_name is a string of a valid exercise.
+    """
+    if not os.path.exists(exercise := f"{TARGET}/{exercise_name}"):
+        os.makedirs(exercise)
+    if not os.path.exists(meta := f"{exercise}/.meta"):
+        os.makedirs(meta)
+    if not os.path.exists(docs := f"{exercise}/.docs"):
+        os.makedirs(docs)
+
+
+def create_meta_config(exercise_name, prob_spec_exercise, author):
```

Comment:
Skimming this but `configlet sync —filepaths —metadata -uye <exercise-slug>` would generate the config json so the only thing needed is update the author or contributor key